mods/tuxemon/maps/spyder_cotton_artshop.tmx

```diff
@@ -173,14 +173,13 @@
   <object id="30" name="Pay" type="event" x="0" y="112" width="16" height="16">
    <properties>
     <property name="act1" value="pathfind player,2,8"/>
-    <property name="act2" value="set_variable paygallery:null"/>
+    <property name="act2" value="transfer_money -,50,spyder_cotton_artshop"/>
```

Comment:
I think this can cause an infinite loop, have you tested it?

Reply:
Exactly, now it's fixed.